examples/train_decoder_only_base.py

```diff
@@ -18,7 +18,10 @@ class TrainDecoderOnlyBase():
 
   def __init__(self):
     self.config = DecoderOnlyConfig()
-    self.batch_size = 16
+    if xr.device_type() == 'NEURON':
+      self.batch_size = 4
+    else:
+      self.batch_size = 16
```

Comment:
if this because one of the dynamo test uses `decoder_only_base`? or you just trying to run this example on your hardware as well.

Reply:
Yes, run on hardware. Is there a different test for hardware?